src/main/java/at/laborg/briss/gui/MergedPanel.java

```diff
@@ -77,7 +82,23 @@
     private final BufferedImage img;
 
     private enum ActionState {
-        NOTHING, DRAWING_NEW_CROP, RESIZING_HOTCORNER_UL, RESIZING_HOTCORNER_LR, MOVE_CROP
+        NOTHING, DRAWING_NEW_CROP, RESIZING_HOTCORNER_UL, RESIZING_HOTCORNER_LR, MOVE_CROP,
+        RESIZING_HOTCORNER_UR, RESIZING_HOTCORNER_LL, RESIZING_RIGHT_EDGE, RESIZING_LEFT_EDGE,
+        RESIZING_UPPER_EDGE, RESIZING_LOWER_EDGE;
+    }
+
+    private static final Map<BiPredicate<DrawableCropRect, Point>, Integer> CURSORS_FROM_CROP_AND_POINT = new HashMap<>();
```

Comment:
In some occasions I haven't been able to see the cursor for some of the edges and it would just show the hand cursor. In fact every time the app starts this behaviour changes.
I suspect that the Map causes the issue here because the entries are distributed differently by the hash function on each start.
In this case however I think we want the order to be fixed and the hand cursor used only at the end if none of the other match. One way to ensure that is to use an Array or ArrayList with a new Class which holds the `BiPredicate<DrawableCropRect, Point>` and `Integer` cursor part.

Reply:
Sure if you like to give it a try, let's go

Reply:
So I guess this change will be part of a new PR?